editor/modes/visual-editor/block.js

```diff
@@ -72,6 +72,8 @@ class VisualEditorBlock extends Component {
 		this.onKeyDown = this.onKeyDown.bind( this );
 		this.onBlockError = this.onBlockError.bind( this );
 		this.insertBlocksAfter = this.insertBlocksAfter.bind( this );
+		this.onTouchStart = this.onTouchStart.bind( this );
+		this.onClick = this.onClick.bind( this );
```

Comment:
What would be the diff between touch start and click here? Might it be better stick to touch events entirely to set `this.hadTouchStart`?

Reply:
I tried onTouchEnd but it fires too soon it needs to be after mousemove